crates/sozo/src/commands/build.rs

```diff
@@ -47,3 +47,21 @@ impl BuildArgs {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use dojo_test_utils::compiler::build_test_config;
+
+    use super::BuildArgs;
+
+    // this adds considerable time to test suite, should this be enabled by default?
```

Comment:
`build.rs` has it but it doesn't generate bindings. since build is done this shouldn't be expensive to do assuming cairo compiler does incremental builds

Reply:
As the plugin system is something that is done post-compilation, what's generated by `build.rs` from `dojo-test-utils` is enough to run tests. But this is done inside the `dojo-bindgen` crate.

Inside `sozo`, it's currently not possible to decouple the build from the plugin system being invoked after compilation. For this reason, if the time to execute the test suite must remain reasonable, we can remove this test for now.

However, when the `dry-run` option will be available, we could imagine `sozo` outputting the migration strategy + the plugins that are expected to run. Like this, `sozo` tests cover the plugin integration, but not the actual bindings generation (which is expected to be tested in `dojo-bindgen`).

Sounds reasonable to you guys?